### ISSUE TITLE
command not found

### ISSUE THREAD

**Issue by bablf** [cdli-gh/CDLI-CoNLL-to-CoNLLU-Converter#16]
after using pip install I cant call cdliconll2conllu. It is installed which I can check with `pip freeze | grep cdli ` and in my PATH variable. 

I get an command not found error.

**Comment by bablf** [comment]
Managed to get it to work with python3.9 and only if I use `python3.9 -m cdliconll2conllu.cli --input_path "./train/cnn_0001.v4_gold_conll" -s "./train/conllu/" `  But I still get an error saying `TypeError: stat: path should be string, bytes, os.PathLike or integer, not tuple`

**Comment by megamattc** [comment]
I currently cannot get the program to run. I execute ```cdliconll2conllu -i input_file.conll``` but it says the command cannot be found.

**Comment by larsgw** [comment]
I think 359b1f2edd22cd5d86e6673e6602e1c608c7a04d should fix it, could you try again?

**Comment by larsgw** [comment]
(by reinstalling)

**Comment by megamattc** [comment]
I reinstalled the package and it does execute, but when I run the command

```
cdliconll2conllu -i inscriptions.conll
```

where ```inscriptions.conll``` is in the same directory, I get a prompt

```
Input path with sytnax: 
```

I don't understand what to do with this.

**Comment by larsgw** [comment]
I only checked whether it was installed, sorry. It looks like some (relatively) recent changes are causing problems, but I don't know what to do about those at the moment. Reverting to 8c7248734beb9b165c08c4e8dfbb3c58c82f0389 worked for me:

    pip3 install git+https://github.com/cdli-gh/CDLI-CoNLL-to-CoNLLU-Converter.git@8c7248734beb9b165c08c4e8dfbb3c58c82f0389

**Comment by megamattc** [comment]
Ok. It now does convert.

However, I did not realize it can only process one text at a time. I fed it a file with many cdliconll texts in it, each prefixed by a comment line with P-number and line numbers, but that all gets collapsed into one text in the output. Since I am trying to convert search results from cdli (which puts many atf transliterations in one file), this is problematic.

**Comment by epageperron** [comment]
I don't think that in our pipeline we have had needed to process multiple texts from one file for this specific conversion; this is why the app cannot handle this. It should be a trivial change to implement. I'll open a separate issue about this.

**Comment by megamattc** [comment]
Hmm. I think this would be an issue for people searching for a group of texts in cdli because the export feature puts all the found texts in one file.

**Comment by epageperron** [comment]
I would presume that people would want to download annotated texts in those formats, not unannotated ones. As for the ATF files, we need the atf texts in one file so editors can easily update multiple texts using grep in a simple text processing software and then re-upload the changes. Uploading atf and cdli-CoNLL annotations in cdli is done with one file. The cdli-CoNLL 2 CoNLL-U converter is used in the backend, with texts sent one by one to the app.   
  
In all cases, we have very few consumers, so I am happy that we adapt to the needs that arise; however, we are understaffed. In the email I just sent you I outlined a potential solution if you are interested.

**Comment by megamattc** [comment]
Ok. Thanks.
